<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_AnsyData</class>
 <widget class="QDialog" name="Dlg_AnsyData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>流量统计</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/res/qrob.png</normaloff>:/res/qrob.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="bg">
     <property name="styleSheet">
      <string notr="true">QFrame#bg{background:white;}
QToolButton{font-size:16px;}
QLabel{font-size:14px;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QCustomPlot" name="plot" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>tcp4:</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/icon_data.svg</normaloff>:/res/icon_data.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lb_tcp4">
           <property name="text">
            <string>1024</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>字节</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>tcp6:</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/icon_data.svg</normaloff>:/res/icon_data.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="lb_tcp6">
           <property name="text">
            <string>1024</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>字节</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>udp4:</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/icon_data.svg</normaloff>:/res/icon_data.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lb_udp4">
           <property name="text">
            <string>1024</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>字节</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string>udp6:</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/icon_data.svg</normaloff>:/res/icon_data.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="lb_udp6">
           <property name="text">
            <string>1024</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>字节</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
